### DISCUSSION TITLE
Drag/move window without decorations?

### DISCUSSION THREAD

**Discussion by awaowouwu** [pthom/hello_imgui#45] [Q&A]
Hello, is it possible to make "backend window" of GLFW draggable for example by grabbing the top "menu bar"?
I'd like to replace OS window decorations with my own, but then I think there's no way to drag the window around.
I don't use fullscreen or docking, I'd just like to render one small window with it.

**Comment by pthom** [comment]
Hello,

See configWindowsMoveFromTitleBarOnly  in [ImGui window params](https://github.com/pthom/hello_imgui/blob/master/src/hello_imgui/hello_imgui_api.md#imgui-window-params)

If false, windows can be dragged from anywhere

**Comment by pthom** [comment]
> Hello,
> 
> See configWindowsMoveFromTitleBarOnly in [ImGui window params](https://github.com/pthom/hello_imgui/blob/master/src/hello_imgui/hello_imgui_api.md#imgui-window-params)
> 
> If false, windows can be dragged from anywhere

Sorry, this works only for inner ImGui windows, not for the main app window.

**Comment by pthom** [comment]
> don't use fullscreen or docking, I'd just like to render one small window with it.

In that case, I don't think of an immediate solution. You may handle this by yourself for example by calling 

```cpp
    auto pos = ImGui::GetMousePos();
    if (pos somewhere near the menu bar)
       if (ImGui::IsMouseDragging(0))
          move the backend window
```
